packages/es-components/src/components/controls/radio-buttons/RadioGroup.js

```diff
@@ -71,4 +71,14 @@ RadioGroup.propTypes = {
   extraContent: PropTypes.node
 };
 
+RadioGroup.defaultProps = {
+  legendContent: null,
+  value: null,
+  hasError: false,
+  disableAllOptions: false,
+  inline: false,
+  onChange: () => {},
```

Comment:
Maybe noop here since we're using it in most other components?

Reply:
Done.